src/main/java/com/malli/main/SeleniumTest.java

```diff
@@ -2,6 +2,7 @@
 
 public class SeleniumTest {
     public static void main(String[] args) {
-        System.out.println("Hellow World");
+
+        System.out.println(" selenium examples");
```

Comment:
update s with  uppercase in selenium word

Reply:
resolved